ketchup/main.py

```diff
@@ -75,46 +77,50 @@ def run():
     except ValueError:
         per_day_limit = None
 
+    total_episodes_per_week = per_day_limit * days_per_week_watching
+
+    weeks_required = math.ceil(episodes / total_episodes_per_week)
+
     # Time to math!
-    watch_schedule = wsb.calculate_watch_schedule(
-        episodes, end_date, start_date, per_day_limit
+    watch_schedule = dc.calculate_watch_schedule(
+        episodes, end_date, start_date, per_day_limit, total_episodes_per_week
```

Comment:
Ah this makes sense, nice.

Reply:
Yeah, more than the per_week_limit as beforehand. It was even confusing me at times when testing the code.

The wording was "How many episodes would you like to watch a week?" and I would put 10, but the calculation is dividing the overall episodes by seven, for days in the week. So the outcome I expected was incorrect.